Makefile

```diff
@@ -75,26 +71,28 @@ UIPATH=$(BUILDPATH)/src/ui
 UINGPATH=$(BUILDPATH)/src/ui_ng
 GOBASEPATH=/go/src/github.com/vmware
 CHECKENVCMD=checkenv.sh
-BASEIMAGE=photon
-COMPILETAG=compile_normal
+
+# parameters
 REGISTRYSERVER=
 REGISTRYPROJECTNAME=vmware
 DEVFLAG=true
 NOTARYFLAG=false
-REGISTRYVERSION=2.6.2-photon
-NGINXVERSION=1.11.13
-PHOTONVERSION=1.0
-NOTARYVERSION=server-0.5.1
-NOTARYSIGNERVERSION=signer-0.5.1
-MARIADBVERSION=10.2.10
+CLAIRFLAG=false
 HTTPPROXY=
 REBUILDCLARITYFLAG=false
 NEWCLARITYVERSION=
+BUILDBIN=false
+MIGRATORFLAG=false
 
-#clair parameters
-CLAIRVERSION=v2.0.1-photon
-CLAIRFLAG=false
-CLAIRDBVERSION=9.6.5-photon
+#versions
+REGISTRYVERSION=v2.6.2
+NGINXVERSION=1.11.13
```

Comment:
The version of nginx, mariadb, postgresql can be misleading because they are pulled from photon's repo.
How about remove their own version and tag the images as `harbor-${VERSIONTAG}`

Reply:
agree. we can just use ${VERSIONTAG}.